.github/DISCUSSION_TEMPLATE/request-help.yml

```diff
@@ -1,4 +1,13 @@
 body:
+  - type: dropdown
```

Comment:
I see you're renaming this file. Did you check if we reference the deleted or renamed file in the code elsewhere?

Reply:
I didn't find any references to either filename, thanks